src/store/actions.ts

```diff
@@ -998,25 +996,17 @@ export const setAddressSuggestions: ActionIF = ({ commit }, addressSuggestions:
   commit('mutateAddressSuggestions', addressSuggestions)
 }
 
-// TODO: Not a real action
-export const fetchCorpNum = async ({ getters }, corpNum: string): Promise<any> => {
+// TODO: Not a real action?
+export const corpNumRequest = async ({ getters }, corpNum: string) => {
```

Comment:
I had renamed this to convey that it was a network action (fetch). Do you really prefer this name?
(Or was there a merge issue that knocked out my changes?)

Reply:
I don't see a similar rename of the code that calls this method so I assume this rename is a merge snafu. Please double check this method's name and the places that call it.

Reply:
kk will do

Reply:
Fixed

Reply:
Ah, reverted. That's fine.